test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/Linux/AcceptanceTestResourceUtilizationLinux.cs

```diff
@@ -37,7 +37,6 @@ public void Adding_Linux_Resource_Utilization_Allows_To_Query_Snapshot_Provider(
     }
 
     [ConditionalFact]
-    [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX, SkipReason = "Want to see if it throws on windows.")]
```

Comment:
is this right? seems it wants to run on Windows. maybe needs to be in a different class/fixture.

Reply:
Ugh... now I have go and read it, it was a mechanical kind of change.
Thanks!